### ISSUE TITLE
Auto popup doesn't often work after updated to v.3.42.0

### ISSUE THREAD

**Issue by Thomasc5** [CanisLupus/swift-selection-search#187]
After SSS for Firefox is updated to v. 3.42.0,  auto popup (i.e. open on text selection) doesn't work intermittently. Reload the page will solve the problem.  

**How to reproduce the problem**
1. Go to any website (e.g. Wikipedia)
2. Middle click 5-6 links successively and let them load in the background
3. Highlight text: auto popup wouldn't work on some pages 
4. Press F5 to reload those pages will make SSS auto popup on text selection work again 

**Anything else you'd like to include**
Earlier SSS versions (e.g. v. 3.41.2) do not have this problem

**System info:**
- Browser: Firefox 
- Browser version: 74.0 (64-bit)
- System: Windows 7 

Thank you

**Comment by CanisLupus** [comment]
Hi @Thomasc5 I must be really quick in this reply but I thank you very much for reporting this issue. I've made a very quick fix and submitted version 3.42.1 to Mozilla. I don't know when it will be live but it should fix this problem. Sorry for this! Later today I'll have more time.

Cheers!
Daniel

**Comment by CanisLupus** [comment]
So, I'm in less of a rush now. :) It also seems the version is live now.

The issue was due to a change I made for the website blocklist feature. The code was always registering the addon's page script with the currently active tab instead of the actual just-opened tab (I hadn't noticed), so tabs opening in the background didn't get the popup. I tested this but apparently I didn't use background tabs for the test.
